src/ProductConstructionService/ProductConstructionService.Api/PcsStartup.cs

```diff
@@ -107,15 +108,19 @@ static PcsStartup()
     /// Registers all necessary services for the Product Construction Service
     /// </summary>
     /// <param name="builder"></param>
-    /// <param name="initializeService">Run service initialization? Currently this just means cloning the VMR</param>
+    /// <param name="isDevelopment">Is this development environment?</param>
     /// <param name="addKeyVault">Use KeyVault for secrets?</param>
     /// <param name="addSwagger">Add Swagger UI?</param>
-    internal static void ConfigurePcs(
+    internal static async Task ConfigurePcs(
+    //internal static void ConfigurePcs(
         this WebApplicationBuilder builder,
-        bool initializeService,
+        bool isDevelopment,
```

Comment:
I think we don't need this. We can query `builder.HostinEnvironment.IsDevelopment`